### ISSUE TITLE
Step by step guide for @davezuko/react-redux-starter-kit

### ISSUE THREAD

**Issue by przeor** [davezuko/react-redux-starter-kit#984]
Hi Guys,

I've open sourced the book that uses this starter kit. It's available online at https://reactjs.co/ 

I would be grateful if you can give me feedback, how I can improve the quality and what is missing in that guide.

Do you find it useful?

Regards,
Kamil

**Comment by grigy** [comment]
Hi Kamil,
Thank you for the book. It's definitely useful.
First impression - the animations are distracting. Also the layout and styles of text could be improved for better readability. More comments after I read the book ;)

**Comment by przeor** [comment]
@grigy do you mean the animations on the top or the one that shows the app dev progress like this one https://reactjs.co/wp-content/react_convention/938_anim.gif ???

**Comment by grigy** [comment]
I mean the ones on the top. Regarding the app dev progress, personally I don't see the need in them. But if you think they help to understand you could make them activated by a click. The text with embedded animation is really hard to follow.

**Comment by przeor** [comment]
thank you @grigy - generally, I've deleted them as you suggest but I will also take a look into Google Analytics on Saturday if the average time page on page is better/same/worse :-) 

My idea behind those gif animations: I thought that those top animations are "kind of a teaser" which may create more interest in the whole book's content. 

We will see in 3 days if the deletion of the top animation has changed the average's user behaviour :-) 

@All I am looking for more feedback so I can improve the experience. I've also created an FB group https://www.facebook.com/groups/reactjs.co/ where I got already some request about potential contents' improvements.

**Comment by DarynHolmes** [comment]
@przeor thanks for the book. It is really helping me!

I like the gif animations, they helped clarify some issues.

Thank you for taking the time to put this together!

**Comment by przeor** [comment]
@DarynHolmes your good words are appreciated. Thank you.

Stay tuned, I plan to expand this book into React-Native (based on the current www.ReactJS.co project structure, but with React-Native in the views) and maybe some backend :-) with GraphQL/Relay? We will see.

If anyone want to join that project, please feel free to ping me :-)

**Comment by haderman** [comment]
@przeor you could use the blueprints with redux-cli to dashboard creation

**Comment by CatPhat** [comment]
@przeor Thank you for taking the time to put this helpful guide together.

One suggestion that could assist with the readability is removing the constant reminders that you can click the images to make them larger.

<img width="957" alt="screen shot 2016-09-26 at 17 08 26" src="https://cloud.githubusercontent.com/assets/6219799/18830688/008ec88a-840c-11e6-9346-68af8e996dbb.png">

**Comment by przeor** [comment]
OK I will remove them. Btw. have you seen the React Native Convention part with the same codebase structure at https://reactjs.co/react-native-convention/ ?

**Comment by davezuko** [comment]
This was added to the README a while ago, so hopefully everybody can find this resource. Thank you again!

**Comment by przeor** [comment]
Hi @davezuko 

I'm looking around and I can't find it in the learning section (and in other places too):
<img width="1039" alt="screen shot 2016-11-04 at 09 47 17" src="https://cloud.githubusercontent.com/assets/5995/19999622/c4247242-a273-11e6-8468-9636986a802d.png">

**Comment by davezuko** [comment]
@przeor swear I added it, maybe it was to the FAQ. Reopening until I add it.

**Comment by przeor** [comment]
@davezuko have you added the tutorial to the readme? We still working on improving it's quality, so it's getting better and better with time :-)

**Comment by NorianNyx** [comment]
@przeor Thanks for the guide! I do have one question though. In your Dashboard module you have the logic for doing things like the reorder/edit in your ACTION_HANDLERS. If I wanted to split this logic off into separate functions, where would be the appropriate place to put them?

**Comment by przeor** [comment]
@NorianNyx if you want create some separate utilities functions then the best location is `src/utils`.

If the separate function is more related to the dashboard only, then I would create a new file under the `routes/dashboard/modules/XYZ.js` or simply put in the same file the functions in your own preferable order.

Does it help @NorianNyx ?
